tools/dump-led-object-paths.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include "utility.hpp"
+
+
+/**
+ * @brief API to dump all the led object paths.
+ */
+void dumpLEDObjectPaths()
+{
+    std::cout << "Dump the objects under LED Group" << std::endl;
+
+    std::vector<std::string> interfaces{"org.freedesktop.DBus.Introspectable"};
```

Comment:
Do we need this interface?

Reply:
yes, When I triggered the patch of NO interface it returned me balank. so i gave this.

Reply:
No, that should not happen, interface parameter is optional. Please re-check.